releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,12 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.0.0-preview4 - September xx, 2023
+
+### New Features
+* Added support for periodically resolving Key Vault references and update the values for associated data items in generated Kubernetes secret accordingly. 

```

Comment:
Added support for periodically resolving Key Vault references to fetch latest version secret and update Kubernetes secret accordingly